docs/MVS_parameters_list.csv

```diff
@@ -45,3 +45,5 @@
 43, None, The type of the component., demand, *demand*, str, None,type_asset,typeasset-label
 44, None," Input the type of OEMOF component. For example, a PV plant would be a source, a solar inverter would be a transformer, etc.  The `type_oemof` will later on be determined through the EPA.", sink, *sink* or *source* or one of the other component classes of OEMOF., str, None,type_oemof,typeoemof-label
 45, None, Unit associated with the capacity of the component.," Storage could have units like kW or kWh, transformer station could have kVA, and so on.", Appropriate scientific unit, str, NA,unit,unit-label
+46, None, The maximum amount of total emissions in the optimized energy system.,100000, None, Alphanumeric,None or kg/a,maximum_emissions,maxemissions-label
+47, None, Emissions per unit of an asset., 14.4, None, Numeric,kgCO2/unit,emission_factor,emissionfactor-label
```

Comment:
```suggestion
47, None, Emissions per unit dispatch or capacity of an asset, 14.4, None, Numeric,kgCO2/unit,emission_factor,emissionfactor-label
```
I am not sure here, but we need to say whether the unit would be a unit dispatch or a unit capacity. I am confused.

The values should be restricted to positive ones

Also: At the end of the PR, please sort the parameters in the csv by their label name, so that we have an alphabetical list again ;)

When defining new parameters, you would actually also define a default value here: https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/utils/constants.py#L243

Reply:
@MaGering if @SabineHaas does sort the `csv` alphabetically, you will get merge conflicts in your PR. Take care when merging!